Library/Homebrew/rubocops/mixins/on_formula_class.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop
+  module Cop
+    # Helper module for checking formula class
+    module FormulaDef
```

Comment:
To me it makes more sense for there to be a `HomebrewCop` base class that other cops inherit from as all will want to use this code.